modules/applications/examples/ui-builder/production/docker-compose.yml

```diff
@@ -5,13 +5,9 @@ services:
     container_name: bonita-ui-builder
     environment:
       BONITA_DEV_MODE: false
-      BONITA_API_URL: http://host.docker.internal:8080/bonita/API
+      BONITA_API_URL: http://bonita-app:8080/bonita/API
       APPSMITH_ENCRYPTION_PASSWORD: <encryption-password>
       APPSMITH_ENCRYPTION_SALT: <encryption-salt>
-    ports:
```

Comment:
**question:** I don't understand why we should remove the expose ports. It will not working ?

Reply:
as ui-builder needs to contact several endpoints to achieve login; Using the port directly will not work correctly ...
The login mechanism will not work if we use directly the UIB port.
So, it is better to expose only the proxy port because all traffic must pass only from it.

Reply:
But, the proxy image needs the exposed ports of other image to access to them, no ?

Reply:
It's no required; as you see i used services names with internal ports
services in the same docker network can communicate using service name and internal ports 😎  🪄

Reply:
It's too magic 😂🤩